changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -1,55 +1,121 @@
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "accessReviewHistoryStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **accessReviewHistoryStatus** enumeration type.",
+          "Target": "accessReviewHistoryStatus"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewHistoryScheduleSettings",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewHistoryScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryScheduleSettings?view=graph-rest-1.0) resource type.",
+          "Target": "accessReviewHistoryScheduleSettings"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewHistoryDefinition",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewHistoryDefinition](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryDefinition?view=graph-rest-1.0) resource",
+          "Target": "accessReviewHistoryDefinition"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewHistoryInstance",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewHistoryInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewHistoryInstance?view=graph-rest-1.0) resource",
+          "Target": "accessReviewHistoryInstance"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Resource",
+          "ChangedApiName": "historyDefinitions",
+          "ChangeType": "Addition",
+          "Description": "Added the **historyDefinitions** relationship to [accessReviewSet](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewSet?view=graph-rest-1.0) resource.",
+          "Target": "accessReviewSet"
+        },
+        {
+          "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+          "ApiChange": "Method",
+          "ChangedApiName": "generateDownloadUri",
+          "ChangeType": "Addition",
+          "Description": "Added the [generateDownloadUri](https://docs.microsoft.com/en-us/graph/api/generateDownloadUri?view=graph-rest-1.0) method to the [generateDownloadUri](https://docs.microsoft.com/en-us/graph/api/generateDownloadUri?view=graph-rest-1.0) resource.",
+          "Target": "generateDownloadUri"
+        }
+      ],
+      "Id": "9d9f39ab-b84b-439f-b0fc-024ad885c1ad",
+      "Cloud": "Prod",
+      "Version": "v1.0",
+      "CreatedDateTime": "2022-01-26T19:09:41.0191537Z",
+      "WorkloadArea": "Microsoft.IdentityGovernance.AccessReviews",
+      "SubArea": ""
```

I understand from the onboarding team that this misrepresentation of WorkloadArea and SubArea is because you haven't defined the correct values in the workload-mapping.json file. Please fix the file so this resolves in future changelogs.
```suggestion
      "WorkloadArea": "Identity and access",
      "SubArea": "Governance"
```

---

Here is the PR to fix the mapping json file: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5737975